static/js/gear_menu.js

```diff
@@ -86,6 +86,9 @@ exports.update_org_settings_menu_item = function () {
 exports.initialize = function () {
     exports.update_org_settings_menu_item();
 
+    $('#gear-menu').on('click', function () {
```

Comment:
Is there a reason we don't just do this as part of the `shown` method, about four lines down, for consistency?

Reply:
I tried this, after your suggestion. But it doesn't seem to work. Here, the shown method is called only the first time when the page loads. Not every time we click(That is what I observed.)